File: mailorder/src/model/DrugDetail.ts
import { DrugLocation } from "./DrugLocation";

export class DrugDetail{
	drugId:string = '';
	drugName:string ='';
	manufacturerName:string = '';
    manufacturedDate :Date = new Date();
	expiryDate :Date = new Date();
    drugLocationList:DrugLocation[] = [];
    
    // constructor(drugId:string, drugName:string, manufacturerName:string, manufacturedDate:Date, expiryDate:Date, drugLocationList:DrugLocation[]){
    //    this.drugId = drugId;
    //    this.drugName = drugName;
    //    this.manufacturerName=manufacturerName;
    //    this.manufacturedDate = manufacturedDate;
    //    this.expiryDate = expiryDate;
    //    this.drugLocationList = drugLocationList;

    // }
}

File: mfpeproject/subscription-microservice/src/main/java/com/pack/pojo/DrugDetail.java
package com.pack.pojo;

import java.util.Date;
import java.util.List;

import io.swagger.annotations.ApiModel;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
@ApiModel(value = "Model object that stores the Package information.")

public class DrugDetail {
	
	
	private String drugId;
	
	private String drugName;
	
	private String manufacturerName;
	private Date manufacturedDate;
	private Date expiryDate;
	
	private List<DrugLocation> drugLocationList;

}
